crocks/Pred.js

```diff
@@ -37,7 +40,7 @@ function Pred(runWith) {
       throw new TypeError('Pred.concat: Pred required')
     }
 
-    return Pred(x => runWith(x) && m.runWith(x))
+    return Pred(x => !!runWith(x) && !!m.runWith(x))
```

Comment:
Good question.
We lock the result of `Pred` to a `Boolean`.
We need to return a `Boolean`.
Now as far as the Error, it is perfectly acceptable to have functions that return non-boolean values and be interpreted as Booleans. So if we were to throw, then that means they would have to ensure that all functions they wrap will need to return a boolean. That means if they have existing functions they want to use as Pred, then they will need to create wrappers.

Does not seem very JS to me to do that.
Thoughts?